docs/static_site/src/_includes/get_started/linux/python/cpu/pip.md

```diff
@@ -9,17 +9,28 @@ page](https://mxnet.apache.org/get_started/download).
 
 Run the following command:
 
-<div class="v1-6-0">
+<div class="v1-7-0">
 {% highlight bash %}
 pip install mxnet
 {% endhighlight %}
 
+Start from 1.7.0 release, MKL-DNN is enabled in pip packages by default. Which are
+optimized for Intel hardware. You can find performance numbers
```

Comment:
Elaborate on the "optimization? For example "oneAPI Deep Neural Network Library (oneDNN) is an open-source cross-platform performance library of basic building blocks for deep learning applications. The library is optimized for Intel Architecture Processors, Intel Processor Graphics and Xe architecture-based Graphics. Support for other architectures such as Arm* 64-bit Architecture (AArch64) and OpenPOWER* Power ISA (PPC64) is experimental." (from the oneDNN repo).

Please mention that the `mxnet-native` release is available which does come without oneDNN

Reply:
Thank you for the valuable inputs, leezu, I will update the description here.

Reply:
BTW, I didn't find available `mxnet-native` pip wheel via `pip install mxnet-native`, is it still WIP or using the different name?

Reply:
yes it's WIP.